server/bin/deploy

```diff
@@ -286,6 +287,22 @@ else
     HOSTEDTEST=""
 fi
 
+
+if [ "$QPID" == "qpid" ]; then
+    info_msg "Enable Qpid"
+    QPID="qpid=yes"
+
+    #Setup testing qpid queue allmsg, this will be needed by Qpid spec tests
+    QPID_KEYS_DIR=$PROJECT_DIR/../gutterball/bin/qpid/keys
+    # Delete the allmsg first, just to make the script indempotent
+    qpid-config -b amqps://localhost:5671 --ssl-certificate $QPID_KEYS_DIR/qpid_ca.crt --ssl-key $QPID_KEYS_DIR/qpid_ca.key --force del queue allmsg || true
```

Comment:
This commands output should be silenced, because it prints confusing error message when the queue does not exist (which is not an error).